lib/improver/cli/__init__.py

```diff
@@ -144,6 +144,28 @@ def inputcube(to_convert):
     return maybe_coerce_with(load_cube, to_convert)
 
 
+@value_converter
+def inputadvection(to_convert):
+    """Loads cube from file or returns passed object.
+
+    Args:
+        to_convert (string or iris.cube.Cube):
+            File name or Cube object.
+
+    Returns:
+        Loaded cube or passed object.
+
+    """
+    from improver.utilities.load import load_cube
+
+    u = maybe_coerce_with(load_cube, to_convert,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable name "u" doesn't conform to snake_case naming style](https://app.codacy.com/app/metoppv_tech/improver/pullRequest?prid=4562406)